iina/MPVController.swift

```diff
@@ -1109,12 +1105,10 @@ class MPVController: NSObject {
 
     case MPV_EVENT_PLAYBACK_RESTART:
       DispatchQueue.main.async { [self] in
-        player.info.isIdle = false
-        player.info.isSeeking = false
         // When playback is paused the display link may be shutdown in order to not waste energy.
```

Comment:
No state update after seeking? This doesn't seem to be correct.

Reply:
Good catch. The `isSeeking` flag needs to be kept separate. It can't be included in state enum. I've pushed an update.